tracking/include/pcl/tracking/impl/pyramidal_klt.hpp

```diff
@@ -250,15 +250,11 @@ pcl::tracking::PyramidalKLTTracker<PointInT, IntensityT>::downsample(
   for (int j = 0; j < height; ++j) {
     int jj = 2 * j;
 ||||||| parent of 332294ef9... run format again, some stuff was missed
```

Comment:
Bad merge here

Reply:
I still don't know what happened there :confused: 
Anyway, the formatting CI now passed :+1: